requirements.txt

```diff
@@ -0,0 +1,5 @@
+opustools-pkg
+pandas
+p_tqdm
+rapidfuzz
+joeynmt
```

Comment:
@jaderabbit and @juliakreutzer is this the right way to install joeynmt?

like `pip install joeynmt`

I remember last time I tried it it gave me headaches and I was forced to install it from Github...

using `pip install git+https://github.com/joeynmt/joeynmt.git`

also it is missing the version...

Reply:
@espoirMur I added it to pypi a few weeks ago so this should be fine now - I still need to update the folder locations later down so we don't need to install joeynmt via cloning it. 

Yes, I should do versions for all the requirements - thanks for the push!

Reply:
if it is on PyPI that is cool....